src/bindgen/ir/enumeration.rs

```diff
@@ -250,8 +252,12 @@ impl Enum {
 
     pub fn load(item: &syn::ItemEnum, mod_cfg: Option<&Cfg>) -> Result<Enum, String> {
         let repr = Repr::load(&item.attrs)?;
-        if repr == Repr::RUST {
-            return Err("Enum not marked with a valid repr(prim) or repr(C).".to_owned());
+        if repr.style == ReprStyle::Rust && repr.ty.is_none() {
+            return Err("Enum is not marked with a valid #[repr(prim)] or #[repr(C)].".to_owned());
+        }
+        // TODO: Implement translation of aligned enums.
```

Comment:
Please file a follow-up issue on this.

This is technically a breaking change too, but before that the layout was wrong so...

Reply:
Filed #432.